tests/test_twitter_backfill.py

```
@@ -64,35 +67,46 @@ def test_zero_inputs_zero_score(self):
         assert b["age"] == 10
         assert b["blue_verified"] == 5
         assert b["legacy_verified"] == 10
         assert score == 50
 
     def test_company_affiliation_adds_10(self):
-        score, b = compute_partial_author_weight({
-            "followers": 1500,
-            "createdAt": _years_ago_iso(4),
-            "affiliatesHighlightedLabel": {"label": {"text": "Anthropic"}},
-        })
+        score, b = compute_partial_author_weight(
```

Ruff (0.15.12)

[warning] Unpacked variable `score` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)